R/entities_objects.R

```diff
@@ -0,0 +1,114 @@
+# Contains functions to parse the objects described here: 
+# https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/entities
+
+# <https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/entities#hashtags>
+hashtags <- function(x) {
+  if (NROW(x) == 0) {
```

Comment:
The use of `NROW()` suggests the input is going to be either a vector or a matrix.

Reply:
Yes, I use it in case the object is a NULL vector.

Reply:
I tried to replace some by is.null but it didn't pass the checks then. So it is a combination of both.